packages/core/src/Stores/client-store.js

```diff
@@ -718,6 +719,10 @@ export default class ClientStore extends BaseStore {
         return this.account_status?.status?.includes('financial_assessment_notification');
     }
 
+    get is_poa_older_than_six_months() {
```

Comment:
I'm not sure that it is correct name. what if the period of expiration will be changed? :)

Reply:
hopefully this would suffice for the next few years 🌏